code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityExtension.java

```diff
@@ -265,11 +270,12 @@ void handleUpdateIdentities(@NonNull final Event event) {
 				LOG_SOURCE,
 				"Failed to update identifiers as no identifiers were found in the event data."
 			);
+			resolver.resolve(state.getIdentityProperties().toXDMData(false));
```

Comment:
unrelated to your change, but looks like toXDMData(allowEmpty) is called with false in all cases, can we remove that param? 😁

Reply:
The iOS code sets the value to true when handling identity and get identifiers requests, so I'll leave this as is for now and update if needed when I work on syncing the iOS and Android implementations.